openassessment/xblock/submission_mixin.py

```diff
@@ -260,12 +272,14 @@ def create_submission(self, student_item_dict, student_sub_data, files_descripti
                         "descriptions {files_descriptions}".format(
                             student_item_dict=student_item_dict,
                             student_sub_data=student_sub_data,
-                            files_descriptions=files_descriptions 
+                            files_descriptions=files_descriptions,
+                            files_names=files_names,
```

Comment:
FYI: Not used.

Reply:
Also, I think this is called for every single file so this should log for one file. My assumption is that this is printing same exception for all FileUploadErrors.